packages/typespec-azure-resource-manager/src/rules/arm-resource-invalid-action-verb.ts

```diff
@@ -2,7 +2,7 @@ import { Operation, createRule } from "@typespec/compiler";
 import { getOperationVerb } from "@typespec/http";
 
 import { getActionDetails } from "@typespec/rest";
-import { isInternalTypeSpec, isSourceOperationResourceManagerInternal } from "./utils.js";
+import { isInternalTypeSpec } from "./utils.js";
 
 export const invalidActionVerbRule = createRule({
```

Comment:
also while we are here lets rename this to match exactly the name of the rule camelCase with `Rule` suffix`

```
armResourceInvalidActionVerbRule
```